### DISCUSSION TITLE
Possible Issue with License Count and Login Limit (firewall, rustdesk login failure)

### DISCUSSION THREAD

**Discussion by isential-gmbh** [rustdesk/rustdesk#7679] [General]
When setting up certain clients (only observed on Windows Server 2022 so far), an error as listed below occurs during login

![image](https://github.com/rustdesk/rustdesk/assets/68383884/8bf9795d-c608-4f7f-b9a8-9dd19612fca6)

I have a license for 10 users. However, there are only two users in the company. We have installed 10 clients so far and logged in to each one. Could it be that only the logins are being counted, and if we exceed 10 logins, this error occurs, even though in our case there are only two 'real' users?

**Comment by rustdesk** [comment]
The error message tells `https://rustdesk.xxx.xxx` can not reached, please make sure `https://rustdesk.xxx.xxx`  can be access on the browser on the same machine.

**Reply by isential-gmbh** [reply]
The address is accessible. Remote access also works on the computer.

Wait! The servers causing trouble are located in the company. And the company is secured by a UTM firewall. I just made an exception for the target IP address, and the error is gone.

Sorry, my fault!

Thanks and regards,

René
